multistream/multistream.go

```diff
@@ -5,7 +5,7 @@ package multistream
 import (
 	"net"
 
-	mss "github.com/whyrusleeping/go-multistream"
+	mss "gx/ipfs/Qmf91yhgRLo2dhhbc5zZ7TxjMaR1oxaWaoc9zRZdi1kU4a/go-multistream"
```

Comment:
When we get to the point where thats actually possible we can discuss switching to that, but for now its non-trivial to be able to go-get gx packages.